openedx/features/philu_utils/utils.py

```diff
@@ -6,12 +9,18 @@
 log = getLogger(__name__)
 
 
-def validate_file_size(file, max_allowed_size):
+def validate_file_size(_file, max_allowed_size):
     """
-    Validate maximum allowed file upload size, raise validation error if file size exceeds.
-    :param max_allowed_size: maximum allowed file size in bytes
+    Validate maximum allowed file size to upload
+
+    Arguments:
+        _file (File/Image): file that needs to be validated
```

Comment:
Rather than writing `(File/Image)`, it is preferred that we write the actual type of object that contains the file/image.